python/sglang/srt/server_args.py

```diff
@@ -2115,6 +2122,11 @@ def add_cli_args(parser: argparse.ArgumentParser):
             action="store_true",
             help="(Deprecated) Enable FlashInfer CUTLASS MoE backend for modelopt_fp4 quant on Blackwell. Supports MoE-EP",
         )
+        parser.add_argument(
+            "--enable-flashinfer-cutedsl-moe",
```

Comment:
i am a bit confused: shouldn't we do `--moe-runner-backend flashinfer_cutedsl` in the first place? why introduce a deprecated flag?

Reply:
I think he does this for consistency w/ other flags